flink-runtime/src/main/java/org/apache/flink/runtime/metrics/ReporterSetup.java

```diff
@@ -81,6 +84,19 @@ public ReporterSetup(final String name, final MetricConfig configuration, Metric
 		return Optional.ofNullable(configuration.getString(ConfigConstants.METRICS_REPORTER_INTERVAL_SUFFIX, null));
 	}
 
+	public Set<String> getExcludedVariables() {
+		String excludedVariablesList = configuration.getString(ConfigConstants.METRICS_REPORTER_EXCLUDED_VARIABLES, null);
+		if (excludedVariablesList == null) {
+			return Collections.emptySet();
+		} else {
+			final Set<String> excludedVariables = new HashSet<>();
+			for (String exclusion : excludedVariablesList.split(";")) {
+				excludedVariables.add(ScopeFormat.asVariable(exclusion));
+			}
+			return Collections.unmodifiableSet(excludedVariables);
+		}
```

Comment:
no, because MetricConfig is based on `Properties` and can't implement `ReadableConfig` since flink-metrics has no dependency on flink-core for historic reasons. It is a truly independent module.